src/reporter/tests/test_different_types.py

```diff
@@ -0,0 +1,364 @@
+from conftest import QL_URL
+from datetime import datetime
+from reporter.tests.utils import delete_test_data, insert_test_data_different_types
+import pytest
+import requests
+import dateutil.parser
+import time
+
```

Comment:
i believe the test are failing because some of the created entities in this test are not deleted properly (cannot say why though). 

a simple solution could be changing type and entity id to something random and not conflicting with other naming. i think @c0c0n3 provided already such an example somewhere in this pr.